### PR TITLE
Newgamequestion

### PR DESCRIPTION
**Pull request by ChristophLeuker**
Thanks for the review on advanced :)

If you play one player to 0. You get a popup which tells you who won and in how many darts.
You can choose to play again (table gets empty) or not (pop up will close, nothing else happen)

### CODE REVIEW

File: components/game501/index.js [hunk 1]
```diff
@@ -42,23 +43,100 @@ const Hint = styled.div`
...
   const [singleResultPlayerTwo, setSingleResultPlayerTwo] = useState([]);
-  console.log(singleResultPlayerOne);
-  console.log(resultPlayerOne);
-  if (resultPlayerOne === 0) {
-    alert("Player One Wins");
+  const [playerOneWins, setPlayerOneWins] = useState(false);
+  const [playerTwoWins, setPlayerTwoWins] = useState(false);
+  const [numberDartsP1, setNumberDartsP1] = useState(0);
+  const [numberDartsP2, setNumberDartsP2] = useState(0);
+  console.log(playerOneName);
```
Comment (mpagels) [inline components/game501/index.js:68]: here is a tiny lonely console.log

**Comment by MariaRiosNavarro** [comment]
The code I have checked seems to be correct, it is true that it was difficult for me to understand what is going on as I don't play darts. I have tried your deployment and you can see that you have to press enter twice to subtract the results and that is what I didn't understand at first. But everything you have implemented is correct on my side.🚀

File: components/newGameQuestion/index.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+              type="button"
+              alt="play again button"
+              onClick={() => window.location.reload()}
+            >
+              Yes
+            </Button>
+            <Button
+              type="button"
+              alt="dont play again button"
+              onClick={handleNoGame}
```
Comment (mpagels) [inline components/newGameQuestion/index.js:75]: A `button` does not need alt `alt` attribute.

**Review by mpagels — APPROVED** [review]
Hey Christop,
a lot of stuff is happening here and I see also a lot of opportunities to refactor some code here in the next steps. Very happy to see that the app is working.

Pls see my notes. Very good work though.

File: components/newGameQuestion/index.js (hunk 2 above)
Comment (ChristophLeuker) [reply components/newGameQuestion/index.js:75]: oh, okay. deleted ;)

File: components/game501/index.js (hunk 1 above)
Comment (ChristophLeuker) [reply components/game501/index.js:68]: deleted

**Comment by ChristophLeuker** [comment]
thanks for the reviews. i know its a lot of code and yes i try to refactor it by time.

**Pull request merged**